graphql-server/pkg/resolvers/proposal.go

```diff
@@ -112,10 +112,10 @@ func (r *proposalTallyResultResolver) OutstandingOption(ctx context.Context, obj
 
 func (r *queryResolver) Proposals(ctx context.Context, input models.QueryProposalsInput) (*models.Connection[models.Proposal], error) {
 	proposalQuery := pkgContext.GetQueriesFromCtx(ctx).Proposal
-	if input.FollowingAddress != nil && *input.FollowingAddress != "" {
-		proposalQuery = proposalQuery.ScopeRelatedAddress(*input.FollowingAddress)
-	} else if input.Filter != nil {
-		proposalQuery = proposalQuery.ScopeProposalStatus((*input.Filter).ToProposalStatus())
+	if input.Address != nil && input.Address.Address != "" {
+		proposalQuery = proposalQuery.ScopeRelatedAddress(input.Address.Address)
```

Comment:
are the is* fields implemented in the previous commit are not used?

Reply:
They are not used in this PR yes, I am implementing them in the following PR.